### DISCUSSION TITLE
進め方について

### DISCUSSION THREAD

**Discussion by haruki256** [gBenkyous/inTime#42] [General]
サーバ側　汎用的に使えるといいよね。。。

**Comment by haruki256** [comment]
クライアント側先に進めて、
サーバ側はそれに合わせて

**Comment by gonzaresu-sengoku** [comment]
1.mockデータを入れて画面作成
⇒まずはデザインから決める

2.サーバは汎用的にデータを返す
⇒条件によって返すデータが変わること

3.フロントで整形してDOM作成

**Comment by haruki256** [comment]
1. サイトのベースデザインを作る
2. ベースデザインをもとに画面を作っていく
3. 汎用的に使えるサーバ側を作る。
